<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸识别</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>141</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="FaceCollectButton">
      <property name="text">
       <string>人脸采集</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FaceTrainButton">
      <property name="text">
       <string>人脸训练</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FaceDetectButton">
      <property name="text">
       <string>人脸识别</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Colse_camera_button">
      <property name="text">
       <string>关闭摄像头</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RecordTimeButton">
      <property name="text">
       <string>打卡</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Camera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>771</width>
     <height>681</height>
    </rect>
   </property>
   <property name="text">
    <string>                                            摄像头</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>771</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
